kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/EncryptionVersion.java

```diff
@@ -33,7 +33,9 @@ public enum EncryptionVersion {
     public static EncryptionVersion fromCode(byte code) {
         switch (code) {
             case 1:
-                return V1;
+                throw new EncryptionException("Alpha encryption version is unsupported, from 0.5.0 we guarantee backwards compatibility");
```

Comment:
How about "Deserialization of EncryptionVersion=1 records is not supported by this version of the encryption filter." ?